electron/downloader.js

```diff
@@ -91,10 +91,11 @@ function deleteIfExists(path) {
 
 export class Downloader {
 
-    constructor(conf, name) {
+    constructor(conf, name, notify) {
         this.config = conf;
         this.name = name;
         this.tmp = null;
+        this.notify = notify;
     }
 
     downloadIfNotExists() {
```

Comment:
this is returning before download is complete. It tries to launch geth while it is still downloading.
```
$ NODE_ENV=development ./node_modules/.bin/electron -r babel-register -r babel-polyfill .
firstRun true
userData:  /Users/elaineo/Library/Application Support/Emerald Wallet
Starting Emerald...
Download geth from https://github.com/ethereumproject/go-ethereum/releases/download/v3.5.0/geth-classic-osx-v3.5.0.zip to /var/folders/7t/qkxlh0mx40d6xvsmg1c07z980000gn/T/download-geth-LVZ5RC/
2017-06-20 21:16:13.288 Electron Helper[22487:11716980] Couldn't set selectedTextBackgroundColor from default ()
key is loaded a20208d4f6d4081c34742a182b6a1d7bfc7bbdef
key is loaded b23dc6dc7634399484deb8a6f36487257419d94c
```

<img width="406" alt="screen shot 2017-06-20 at 9 10 54 pm" src="https://user-images.githubusercontent.com/2349487/27366942-5371ea86-55fe-11e7-80f3-25325398518b.png">